client/my-sites/themes/theme-showcase.jsx

```diff
@@ -201,9 +203,14 @@ const ThemeShowcase = React.createClass( {
 } );
 
 export default connect(
-	( state, { siteId } ) => ( {
-		isLoggedIn: !! getCurrentUserId( state ),
-		siteSlug: getSiteSlug( state, siteId ),
-		isJetpack: isJetpackSite( state, siteId ),
-	} )
+	( state, { siteId } ) => {
+		const isRequestingSite = hasSiteActiveQueryRequests( state, siteId );
+		const isRequestingWpcom = hasSiteActiveQueryRequests( state, 'wpcom' );
```

Comment:
Can't we use any of these [existing](https://github.com/Automattic/wp-calypso/blob/8c4345bc104b40c510526bb1894aed2157cc3de0/client/state/themes/selectors.js#L150L162) [selectors](https://github.com/Automattic/wp-calypso/blob/8c4345bc104b40c510526bb1894aed2157cc3de0/client/state/themes/selectors.js#L255-L285)?